tutorials/R/api-token/api-token.Rmd

```diff
@@ -0,0 +1,266 @@
+token before, you may notice this is faster!
+
+This format applies to all `neonUtilities` functions that involve 
+downloading data or otherwise accessing the API; you can use the 
+`token` input with all of them. For example, when downloading 
+remote sensing data:
+
+```{r getAOP, eval=FALSE, comment=NA}
+
+chm <- byTileAOP(dpID="DP3.30015.001", site="WREF", 
+                 year=2017, check.size=F,
+                 easting=c(571000,578000), 
+                 northing=c(5079000,5080000), 
+                 savepath="/data",
+                 token=NEON_TOKEN)
+
+```
+
+## Token management for open code
+
+Your API token is unique to your account, so don't share it!
+
+If you're writing code that will be available publicly, such as in a GitHub 
+repository, you can't include the line of code above where we assigned 
+your token to `NEON_TOKEN`, since your token is fully visible in the code 
+there. Instead, you'll need to save your token locally, and pull it into your 
+code without displaying it. There are a few ways to do this, we'll show 
+two options here.
+
+* Option 1: Save the token in a local file, and `source()` that file at the 
+start of every script. This is fairly simple but requires a line of code in 
+every script.
+
+* Option 2: Add the token to a `.Renviron` file to create an environment 
+variable that gets loaded when you open RStudio. This is a little harder 
+to set up initially, and you must be using RStudio for it to work, but 
+once it's done, it's done globally, and it will work in every script you 
+run.
+
+### Option 1: Save token in a local file
+
+Open a new, empty R script. Put a single line of code in the script:
+
+```{r scriptToSource, eval=FALSE, comment=NA}
+
+NEON_TOKEN <- "PASTE YOUR TOKEN HERE"
+
+```
+
+Save this file in a logical place on your machine, somewhere that won't be 
+visible publicly. Let's call the file `neon_token_source.R`, and let's say 
+you've saved it to the `/data` folder on your machine. Then, at the start of 
+every script where you're going to use the NEON API, you would run this line 
+of code:
+
+```{r source, eval=FALSE, comment=NA}
+
+source("/data/neon_token_source.R")
+
+```
+
+Then you'll be able to use `token=NEON_TOKEN` when you run `neonUtilities` 
+functions, and you can share your code without accidentally sharing your 
+token.
+
+### Option 2: Save token to the RStudio environment
+
+To create a persistent environment variable, we use a `.Renviron` file. 
+Before creating a file, check which directory RStudio is using as your home 
+directory:
+
+```{r getdir, eval=FALSE, comment=NA}
+
+# For Windows:
+Sys.getenv("R_USER")
+
+# For Mac/Linux:
+Sys.getenv("HOME")
+
+```
+
+Check the home directory to see if you already have a `.Renviron` file, **using 
+the file browse pane in RStudio**. Files that begin with `.` are hidden by 
+default, but RStudio recognizes files that begin with `.R` and displays them.
+
```

Comment:
While I don't think directions need to be given, I think we should at least acknowledge that this can be done through command line or finder if users have hidden files turned on.  As written it sounds like the only way to do this is through RStudio.